FiiPrezent/Services/EventsService.cs

```diff
@@ -17,22 +17,35 @@ public class EventsService
             }
         };
 
-        public Event RegisterParticipant(string verificationCode, string participantName)
+        public void InitializeEvents()
```

Comment:
Using static variables if the real problem here. We'll find a more elegant solution on the next session.